recipes/caf/all/conanfile.py

```diff
@@ -0,0 +1,142 @@
+import os
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+from conans.tools import Version
+
+
+class CAFConan(ConanFile):
+    name = "caf"
+    description = "An open source implementation of the Actor Model in C++"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/actor-framework/actor-framework"
+    topics = "conan", "caf", "actor-framework", "actor-model", "pattern-matching", "actors"
+    license = "BSD-3-Clause", "BSL-1.0"
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    generators = "cmake", "cmake_find_package"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "log_level": ["ERROR", "WARNING", "INFO", "DEBUG", "TRACE", "QUIET"],
```

Comment:
don't we want all lower case in options and options values (except fPIC)?

Reply:
And `True` and `False`? :)  Those are the values used by the library itself, not enums.  We could make them lower case but then we'd need to convert them back when setting the CMake def.

Reply:
`True` and `False` are boolean values not string ;) Yes a conversion property would be nice.

Reply:
Ok - they are lower case now